models/src_iterable.yml

```diff
@@ -0,0 +1,414 @@
+        columns:
+          - name: id
+            description: Unique identifer of the message type
+          - name: name
+            description: User defined name of the message type
+          - name: channel_id
+            description: Channel that this message type belongs to
+
+      - name: template_history
+        description: Table containing unique templates used across campaigns.
+        columns:
+          - name: id
+            description: Unique identifer of the template
+          - name: name
+            description: User defined name of the template
+          - name: template_type
+            description: The type of the template
+          - name: created_at
+            description: Creation timestamp as epoch time in milliseconds
+          - name: client_template_id
+            description: Reference to the client template
+          - name: creator_user_id
+            description: Reference to the user who created the template
+          - name: message_type_id
+            description: Reference to the message type associated with the template
+          - name: updated_at
+            description: Last update timestamp as epoch time in milliseconds
+
+      - name: user_history
+        description: Table containing individual user history information.
+        columns:
+          - name: email
+            description: Unique identifier of the user as well as their email
+          - name: user_id
+            description: User defined identifier of the user. This is not a unique field
+          - name: first_name
+            description: First name of the user
+          - name: last_name
+            description: Last name of the user
+          - name: email_list_ids
+            description: Array of references to the lists associated with the user
+          - name: signup_date
+            description: Timestamp as epoch time in milliseconds of when the user signed up
+          - name: signup_source
+            description: Source where the user signed up
+          - name: updated_at
+            description: Last update timestamp as epoch time in milliseconds
+
+      - name: user_device_history
+        description: Table capturing the history of devices associated with in-app and push notifications to a user.
+        meta:
+          is_enabled: "{{ var('iterable__using_user_device_history', true) }}"
+        columns:
+          - name: application_name
+            description: >
+              The Iterable push integration to associate with the device token. Usually matches the package name (or bundle ID) 
+              of the relevant mobile app. 
+          - name: email
+            description: User email associated with the device. 
+          - name: endpoint_enabled
+            description: Boolean that is true if the user has not disabled Push notifications and their endpoint remains enabled.
+          - name: identifier_for_vendor # always null in our data
```

Comment:
Can we remove these comments throughout the yml? I believe I added these here for my ref but would be good to remove to prevent any confusion.

Reply:
yep!